web_external/extra/documents/Terms of Use.txt

```diff
@@ -0,0 +1,124 @@
+International Skin Imaging Collaboration (�ISIC�)
```

Comment:
I think the curly quotes in the original document aren't being translated to straight quotes here; this also seems to be the case on the `legal/terms_of_use.html` page.

I think we should fix it here, just because it will be more apparent to users who download this and open it in notepad. The `legal/terms_of_use.html` can wait until we do a proper style pass.